content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -35,20 +35,22 @@ registry is [registry.ci.openshift.org](https://registry.ci.openshift.org).
 
 ## How do I log in to pull images that require authentication?
 
-All registries are the internal OpenShift image registry for the cluster they reside on, so authenticating to the registry
-requires authentication to the cluster that hosts it. Once logged in to the OpenShift cluster, the `oc` CLI can be used to
-authenticate to the registry in question. For example, for the [registry.build01.ci.openshift.org](https://registry.build01.ci.openshift.org)
-registry, the cluster is `build01`. Using the [the list of clusters](/docs/getting-started/useful-links/#clusters),
-the console URL for this cluster is found to be [console.build01.ci.openshift.org](https://console.build01.ci.openshift.org).
+If you want to e.g. download pre-release images from [the release page](https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/),
+then you will need to authenticate.
+
+The [app.ci cluster](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com) is the backend for
+`registry.ci.openshift.org`.  The canonical console URL is maintained in the [the list of clusters](/docs/getting-started/useful-links/#clusters).
```

Comment:
This basically duplicates information from the "Summary of Available Clusters" above. The previous form of the paragraph reads clearer to me, personally. I think we should point the the `app.ci` cluster in the text b/c authenticating against a build farm should be a less common case, but overall the doc makes more sense to me in the previous form, sorry.

Reply:
OK I reworked this to mostly keep the same flow and just s/build01/app.ci/ but also clarified things slightly.